cdn/dev/keyboard-search/install.js

```diff
@@ -71,6 +78,16 @@ function startAfterPageLoad_Windows(data) {
   }, 10);
 }
 
+function isLegacyFirefoxVersion() {
+  const browser = document.documentElement.getAttribute('data-browser');
+  if(browser != "Firefox") return false;
+
+  var match = /Firefox\/(\d+\.\d+)/.exec(navigator.userAgent);
```

Comment:
Could the regex determine the FF major version number, and then just compare with integer 87 in l.88?

Reply:
it could, but why?

Reply:
I thought it would simplify the version comparison, but nvm